wixcode-users/wix-users.service.json

```diff
@@ -846,10 +953,161 @@
+                ">",
+                "> + The APIs in `wix-users` can only be used once the page has loaded. Therefore,",
+                "  you must use them in code that is contained in or is called from the",
+                "  [onReady()]($w.html#onReady) event handler or any element event handler.",
+                "> ",
+                "> + Cookie consent banners are only available on the published version of your site." ],
+            "links": [],
+            "examples":
+              [ { "title": "Reset a visitor's consent policy",
+                  "body":
+                    [ "import { resetConsentPolicy } from 'wix-users';",
+                      "",
+                      "// ...",
+                      "",
+                      "resetConsentPolicy()",
+                      "  .then( (policy) => {",
+                      "    console.log(\"The policy is now set to the default site policy.\");",
+                      "  } );" ],
+                  "extra":
+                    {  } } ],
+            "extra":
+              {  } },
+        "extra":
+          {  } },
+      { "name": "setConsentPolicy",
+        "labels": [],
+        "nameParams": [],
+        "params":
+          [ { "name": "policy",
+              "type": "wix-users.Policy",
+              "doc": "The object representing the cookies of the user's consent policy." } ],
+        "ret":
+          { "type":
+              { "name": "Promise",
+                "typeParams":
+                  [ "wix-users.PolicyDetails" ] },
+            "doc": "Fulfilled - All the details of the user's consent policy, including if the the current policy is the default site policy and the policy's allowed cookies.\nRejected - Error message." },
+        "locations":
+          [ { "lineno": 569,
+              "filename": "users.js" } ],
+        "docs":
+          { "summary": "Sets the current user's consent policy regarding allowed cookies and data transfer \n to third parties, such as for GDPR or CCPA purposes.",
+            "description":
+              [ "The `setConsentPolicy()` function returns a Promise that resolves to the",
+                "consent policy details of the user.",
+                "",
+                "You can use the [`onConsentPolicyChanged()`](#onConsentPolicyChanged) event to listen for ",
+                "changes made when a visitor changes their consent policy with `setConsentPolicy()`. Handle ",
+                "the policy change accordingly in the event handler's [ConsentPolicyChangedHandler callback function](#ConsentPolicyChangedHandler). ",
+                "Changes to the consent policy take affect after the page is refreshed.  ",
+                "",
+                "The consent policy helps you comply with [GDPR](https://support.wix.com/en/article/preparing-your-wix-site-for-the-gdpr) and [CCPA](https://support.wix.com/en/article/preparing-your-wix-site-for-ccpa) regulations and lets your users ",
```

Comment:
The transition from "you" in the first sentence to "site owner" in the second sentence implies that they are not the same person. Usually they are the same person unless there's a contributor coding the site. Maybe change second sentence to "You set up the site consent.... 
And then add another sentence or note stating that only site owners can set up the policies etc.